vsintegration/src/FSharp.VS.FSI/FSharp.VS.FSI.Attributes.fs

```diff
@@ -0,0 +1,12 @@
+// Copyright (c) Microsoft Corporation.  All Rights Reserved.  See License.txt in the project root for license information.
+
+namespace Microsoft.VisualStudio.FSharp.Interactive
+
+type DisplayNameAttribute(resName) =
```

Comment:
Can we give these a different name? It's confusing looking at `fsiLanguageService.fs` that they don't point to the System.ComponentModel attributes.

Reply:
I'm in favor of renaming them, but to keep the scope of this PR small (e.g., code generation for resources) I'd like to leave the names as they stand.

Reply:
@Brett: perhaps ResourceCaregoryAttribute, ResourceDescriptionAttribute and ResourceDisplayNameAttribute.